inst/pages/98_exercises.qmd

```diff
@@ -1190,31 +1187,31 @@ PCoA object).
 
 ```{r}
 # Load necessary libraries
-library(SummarizedExperiment)
-library(pheatmap)
+library(TreeSummarizedExperiment)
```

Comment:
Is mia already loaded (I believe it is)? It loads automatically TreeSE so this line can be removed

Reply:
Sure, noted :))

Reply:
@jkc9886 have you fixed this now? if yes, kindly resolve the comment